core/src/main/java/hudson/cli/ConnectNodeCommand.java

```diff
@@ -62,7 +62,7 @@ protected int run() throws Exception {
         boolean errorOccurred = false;
         final Jenkins jenkins = Jenkins.getActiveInstance();
```

Comment:
Is there a reason not to update this (and other usages of `Jenkins#getActiveInstance`) to `Jenkins#get`? Seems like you might as well do it at the same time as updating `Jenkins#getInstance`.